src/app/(auth)/register/page.tsx

```diff
@@ -0,0 +1,132 @@
+'use client';
+import { FormikHelpers, useFormik } from 'formik';
+import * as yup from 'yup';
+import AuthForm from '@/components/organism/auth-form';
+import { useRouter } from 'next/navigation';
+import { translate } from '@/translate';
+import { register } from '@/models/api/register';
+
+const validationSchema = yup.object().shape({
+  email: yup.string().email().required(),
+  password: yup.string().required(),
+  passwordConfirmation: yup.string().required(),
+});
+
+type RegisterFormValues = yup.InferType<typeof validationSchema>;
+
+const initialValues: RegisterFormValues = {
+  email: '',
+  password: '',
+  passwordConfirmation: '',
```

Comment:
Suggestion: We need the firstname and lastname field, can we add it if it's possible?

Reply:
yes, i didn't added it initially because it wasn't in the figma, I'll added then